internal/ethapi/api.go

```diff
@@ -1050,6 +1050,23 @@ func DoEstimateGas(ctx context.Context, b Backend, args CallArgs, blockNrOrHash
 			hi = mid
 		}
 	}
+
+	// Fudging to account for gas required to verify signatures + pass around data.
+	// Specifically, this line accounts for the fact that there's a bit of computation performed in
+	// a "real" transaction that won't be covered by an eth_call:
+	//  1. Going into the OVM_SequencerEntrypoint.
+	//  2. Going into the OVM_ProxyEOA + OVM_ECDSAContractAccount.
```

Comment:
Why do all of the work above if we are just going to modify `hi` anyways?

Reply:
The work above gives us a good estimate of the gas usage, it's just "low" by a pretty predictable amount. Note that we're tweaking it (by adding a fudge factor) rather than completely ignoring the previously done work.